frontend/scripts/nlp-algorithm/NLP.py

```diff
@@ -0,0 +1,254 @@
+
+    for items in sentList:
+        first_words.append(items.split(' ')[0])
+        type_of_ending.append(items[-1][-1])
+
+# validatin of words
+
+    for item in type_of_ending:
+        if item == '.':
+            type_sentence.append('Sentence')
+        elif item == '!':
+            type_sentence.append('Exclamatory')
+        elif item == '?':
+            type_sentence.append('Interrogative')
+        else:
+            type_sentence.append('Unknown ')
+    for items in first_words:
+        if items in wh_question:
+            list_of_question_words.append('WH question')
+        elif items in yN_question:
+            list_of_question_words.append('Y/N question')
+# Here it shows all the sentences with their type
+
+    for i in range(len(type_of_ending)):
+        print('Sentence #', str(i+1), 'Type:', type_sentence[i])
+        if type_sentence[i] == 'Unknown or Numbers':
+            print("This is the type that we did not catch it: ", type_of_ending[i])
+    for i in range(len(list_of_question_words)):
+        print('Question #', str(i+1), 'Type:', list_of_question_words[i])
+
+    countQuestionWords = Counter(list_of_question_words)
+    print('Summary of question words: ', countQuestionWords)
+
+# cleanup the text and make it as words in a list
+
+    cleanedText = myFile.translate(str.maketrans('', '', string.punctuation))
+    tokenizedList = word_tokenize(cleanedText, "english")
+
+# creating a list of stop words in the feedback
+
+    listOfWords = [
+        item for item in tokenizedList if item in stopwords.words('english')]
+    countstopwords = Counter(listOfWords)
+
+# creating a list of words in the feedback excluded all the stop words
+
+    finalList = [
+        item for item in tokenizedList if item not in stopwords.words('english')]
+
+    count = 0
+    totalCount = 0
+    itemsCount = 0
+    for item in tokenizedList:
+        itemsCount += 1
+        if item in stopwords.words('english'):
+            count += 1
+
+# creating a list of emotion words and the words that have emotion according to our pre defined list
+
+    emotionList = []
+    wordList = []
+
+# list of words with their emothions
+
+    with open('emotions.txt', 'r') as file:
+        for line in file:
+            clearLine = line.replace('\n', '').replace(
+                ',', '').replace("'", '').replace(' ', '').strip()
+            word, emotion = clearLine.split(':')
+
+            if word in finalList:
+                emotionList.append(emotion)
+                wordList.append(word)
+            countEmotions = Counter(emotionList)
+            countwords = Counter(finalList)
+    totalCount = count+len(set(tokenizedList) -
+                           set(wordList)-set(stopwords.words('english')))
+
+# creating a blob text
+
+    blob = TextBlob(myFile)
+    wordCount = blob.word_counts
+
+# make all the typos correct
+
+    # correctmyFile = blob.correct()
```

Comment:
This doesn't appear to be a comment, it looks like old code. Should it be deleted?